### DISCUSSION TITLE
Volume slider doesn’t work

### DISCUSSION THREAD

**Discussion by papabak** [music-assistant/support#2144] [Q&A] [answered]
I have the lateste MA beta 139 installed today. I noticed the volume slider doesn't work. The sluder can be moved but the volume doesn't change. 
I had this error before but it was solved in previous versions. The error seems to have returned in this version. Can anyone acknowledge this?
My player is an Denon AVR-X3000. Don't know if that is relevant for the solution.

**Comment by sp0ngeboobs** [comment]
I have the same problem and created a ticket, but it was not resolved. https://github.com/music-assistant/hass-music-assistant/issues/2127

**Reply by OzGav** [reply]
Actually you said it was not a MA problem. Is it DLNA @papabak ?

**Comment by papabak** [comment]
Hi  OzGav,

When I open the Denon AVR X3000 entity (from within MA) and move the volume
slider there then the volume level changes accordingly. So I do think the
problem is (again) in MA.
I think this connection is DLNA, though I have two Denon avr X300 entities,
one for DLNA and one for Airplay. I can’t see which protocol is active as
both entities got the same name in HA.

Op zo 21 apr 2024 om 17:14 schreef OzGav ***@***.***>

**Comment by OzGav** [comment]
How have the Denon been added? It sounds like you are using the HA plugin to connect to them, if so why aren’t you connecting to them directly from MA?

**Comment by papabak** [comment]
The Denon is an integration I had long before MA.  MA has worked fine with
the Denon integration except for a few versions of MA where the volume
slider doesn’t work. The problem must be familiar as it was solved more
than once.

Best regards, Dirk

Op ma 22 apr 2024 om 00:23 schreef OzGav ***@***.***>

**Reply by OzGav** [reply, marked as answer]
You will get a better experience if you use the native provider if you can. So if the receiver has DLNA, AirPlay or Chromecast you should connect directly to it via MA instead of going through HA. (Unfortunately your jpeg didn’t come through the emailed response) Having said that we will look at the volume slider as it should work.

**Comment by papabak** [comment]
By the way, this is what I have available in MA:
See jpeg
________________________________
Van: OzGav ***@***.***>
Verzonden: Monday, April 22, 2024 12:23:34 AM
Aan: music-assistant/hass-music-assistant ***@***.***>
CC: papabak ***@***.***>; Mention ***@***.***>
Onderwerp: Re: [music-assistant/hass-music-assistant] Volume slider doesn’t work (Discussion #2144)


How have the Dennis been added? It sounds like you are using the HA plugin to connect to them, if so why aren’t you connecting to them directly from MA?

**Comment by madbrain76** [comment]
I have the exact opposite problem.

https://github.com/music-assistant/hass-music-assistant/issues/2514
